VTEX - Pick and Pack API.json

```diff
@@ -0,0 +1,4196 @@
+                                        }
+                                    }
+                                },
+                                "example": {
+                                    "message": "success",
+                                    "data": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJob3N0bmFtZSI6Imxhc3RtaWxlIiwidXNlcm5hbWUiOiJsYXN0bWlsZS5jcmlzdGlhbnJvbWVyb3Z0ZXhjb20iLCJpYXQiOjE3Mzk0NDkyNTMsImV4cCI6MTczOTQ1MDE1M30.lkb1e5XDuUBddthwaxrvi_Ub0YBEAEessJy_bvzb-do"
+                                }
+                            }
+                        }
+                    }
+                },
+                "servers": [
+                    {
+                        "url": "https://auth.pickingnpacking.com/prod/{environment}",
+                        "variables": {
+                            "environment": {
+                                "default": "DefaultParameterValue"
+                            }
+                        }
+                    }
+                ]
+            }
+        },
+        "/orders/{orderId}": {
+            "get": {
+                "tags": [
+                    "Orders"
+                ],
+                "summary": "Get order by ID",
+                "description": "Retrieves order details by its ID.\n\n>ℹ️ The response is cached for 30 seconds based on the `orderId` parameter. The endpoint supports up to 30 requests per second, with a maximum of 100 concurrent requests.\r\n\r\n## Permissions\r\n\r\nAny user or [application key](https://developers.vtex.com/docs/guides/api-authentication-using-application-keys) must have at least one of the appropriate [License Manager resources](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3) to be able to successfully run this request. Otherwise they will receive a status code `403` error. These are the applicable resources for this endpoint:\r\n\r\n| **Product** | **Category** | **Resource** |\r\n| --------------- | ----------------- | ----------------- |\r\n| OMS | OMS access | **View order** |\r\n| OMS | OMS access | **Only show orders created by the user (via call center)** |\r\n| Catalog | Telesales | **Assisted Sales** |\r\n\r\nThere are no applicable [predefined roles](https://help.vtex.com/en/tutorial/predefined-roles--jGDurZKJHvHJS13LnO7Dy) for this resource list. You must [create a custom role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#creating-a-role) and add at least one of the resources above in order to use this endpoint. To learn more about machine authentication at VTEX, see [Authentication overview](https://developers.vtex.com/docs/guides/authentication).\r\n\r\n>\u2757 To prevent integrations from having excessive permissions, consider the [best practices for managing app keys](https://help.vtex.com/en/tutorial/best-practices-application-keys--7b6nD1VMHa49aI5brlOvJm) when assigning License Manager roles to integrations.",

```

Comment:
Yes, there are aplicable predefined roles. Please fix the text, including the apropriate predefined roles.

Reply:
This applies to all endpoints except /token